apps/useradmin/src/useradmin/management/commands/promote_to_superuser.py

```diff
@@ -0,0 +1,49 @@
+import os
+import sys
+import time
+import datetime
+import re
+import logging
+
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.auth.models import User
+
+import desktop.conf
+
+if sys.version_info[0] > 2:
+  from django.utils.translation import gettext_lazy as _t, gettext as _
+else:
+  from django.utils.translation import ugettext_lazy as _t, ugettext as _
+
+LOG = logging.getLogger(__name__)
+
+class Command(BaseCommand):
+    """
+    Handler for promoting a user to superuser
+    """    
+    def add_arguments(self, parser):
+        parser.add_argument("--usernames", help=_t("User(s) to promote to superuser."), nargs='+', action="store",required=True)
+    
+    def handle(self, *args, **options):
+        if options["usernames"]:
```

Comment:
Nit: What if `options` does not have `usernames` key? Try reading about `.get()` and how it can help here to make this check safer.